docs/Versions.asciidoc

```diff
@@ -73,3 +73,11 @@ Shared attribute values are pulled from elastic/docs
 ///////
 
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+///////
+APM does not build n.x documentation. Links from .x branches should point to master instead
+///////
+ifeval::[{source_branch} == 7.x]
```

Comment:
Hmm, this worked when I tested locally but doesn't seem to resolve correctly in the ci check.